lib/delta_tables_generator.rb

```diff
@@ -0,0 +1,59 @@
+require 'logger'
+
+require 'active_support/notifications'
+require 'active_support/log_subscriber'
+
+module DeltaTablesGenerator
+  extend self
+
+  delegate :instrument, :subscribe, to: ActiveSupport::Notifications
+
+  # Starts the generation of the deltas data.
+  def generate(day: Date.current)
+    DeltaTablesBackend.with_redis_lock do
+      instrument('generate.delta_tables_generator', day: day) do
+        begin
+          [
+            CommodityCodeEndDated,
+            CommodityCodeStarted,
+            CommodityCodeDescriptionChanged,
+            MeasureEndDated,
+            MeasureStarted,
+            MeasureDeleted,
+            MeasureCreatedOrUpdated,
+          ].map do |importer|
+            importer.perform_import(day: day)
+          end
+          return true
```

Comment:
If this returns a boolean value, maybe it should do the same in the rescue for consistency? In this case the ruby method name convention would be `generate?`. Does this need to return a boolean value?

Reply:
It actually doesn't need to return anything at all, and it's definitely not a question mark method, since the return value is not the purpose of the computation.
The `true` was a mean to:
1. signal that the execution went OK, and 
2. override the fact that by default it would return the array, which is exactly as useless anyway.

Would returning `nil` on success (and raising if there are issues) be a good solution in your view?